e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/remove-fees-from-payment.spec.js

```diff
@@ -71,29 +73,33 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can remove fees from payments`, () =>
     pages.utilisationReportEditPaymentPage.clickRemoveSelectedPaymentsButton();
     pages.utilisationReportEditPaymentPage.errorSummary().contains('Select fee or fees to remove from the payment');
 
-    feeRecordIds.forEach((id) => {
-      cy.get(`[type="checkbox"][id="feeRecordId-${id}"]`).check();
+    feeRecordIds.forEach((feeRecordId) => {
+      getFeeRecordCheckbox(feeRecordId).check();
     });
     pages.utilisationReportEditPaymentPage.clickRemoveSelectedPaymentsButton();
     pages.utilisationReportEditPaymentPage.errorSummary().contains('You cannot remove all the fees. Delete the payment instead.');
-    feeRecordIds.forEach((id) => {
-      cy.get(`[type="checkbox"][id="feeRecordId-${id}"]`).should('be.checked');
+    feeRecordIds.forEach((feeRecordId) => {
+      getFeeRecordCheckbox(feeRecordId).should('be.checked');
     });
   });
 
   it('should remove the selected fees from the payment after clicking the remove selected fees button', () => {
-    const feeRecordId = feeRecordIds[0];
+    const getFeeRecordRow = (feeRecordId) => cy.get(`tbody tr[data-cy="fee-record-details-table-row--feeRecordId-${feeRecordId}"]`);
+    const feeRecordIdToRemove = feeRecordIds[0];
+    const otherFeeRecordIds = feeRecordIds.slice(1);
```

Comment:
Nit but you can write this as `const [feeRecordIdToRemove, ...otherFeeRecordIds] = feeRecordIds`

Reply:
Nice, missed that thanks for spotting - just pushed as fa4a32d841055b5a38a13bc8328c46a83b05cf36